app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def date_to_string(date)
```

Comment:
Looking ahead, since we will need another helper that converts dates to mm/yyyy, can this method name be more explicit?